sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -156,51 +189,64 @@ information and examples.
 
 .. _userguide_plotting_color_bars:
 
-Color Bars
+Color bars
 ----------
 
-A |ColorBar| can be created using a |ColorMapper| instance, which
-contains a color palette. Both on- and off-plot color bars are
-supported; the desired location can be specified when adding the
-|ColorBar| to the plot.
+To create a |ColorBar|, use an instance of |ColorMapper| containing a color
+palette.
 
-.. note::
-    This example depends on the open-source NumPy library in order to
-    generate demonstration data.
+Color bars can be located inside as well as left, right, below or above the
+plot. Specify the location of a color bar when adding the |ColorBar| object to
+the plot using the :func:`~bokeh.models.plots.Plot.add_layout` method.
 
 .. bokeh-plot:: docs/user_guide/examples/plotting_color_bars.py
     :source-position: above
 
+.. note::
+    This example depends on the open-source NumPy library to generate
+    sample data.
+
 .. _userguide_plotting_arrows:
 
 Arrows
 ------
 
-|Arrow| annotations can be used to connect glyphs and label annotations or
-to simply highlight plot regions. Arrows are compound annotations, meaning
-that their ``start`` and ``end`` attributes are themselves other |ArrowHead|
-annotations. By default, the |Arrow| annotation is one-sided with the ``end``
-set as an ``OpenHead``-type arrowhead (an open-backed wedge style) and the
-``start`` property set to ``None``. Double-sided arrows can be created by
-setting both the ``start`` and ``end`` properties as appropriate |ArrowHead|
-subclass instances.
+You can use |Arrow| annotations to connect glyphs and label annotations. Arrows
+can also help highlight plot regions.
+
+Arrows are compound annotations. This means that they use additional |ArrowHead|
+objects as their ``start`` and ``end``. By default, the |Arrow| annotation is a
+one-sided arrow: The ``end`` property is set to an ``OpenHead``-type arrowhead
+(looking like an open-backed wedge style). The ``start`` property is set to
+``None``. If you want to create double-sided arrows, set both the ``start`` and
+``end`` properties to one of the available arrowheads.
 
-Arrows have standard line properties to set the color and appearance of the
-arrow shaft:
+The available arrowheads are:
+
+.. bokeh-plot:: docs/user_guide/examples/annotations_arrowheads.py
```

Comment:
This little script demonstrating the available arrow heads was originally part of styling.rst - but I believe it makes more sense to have this information right here in this section about arrows.